src/lab4.py

```diff
@@ -0,0 +1,49 @@
+import textwrap
+
+class laptop:
```

Comment:
Rename class with a capital letter (PEP8)

Reply:
class cant be named with the capital latter becouse it stops being a function

Reply:
class is not a fuction ,its a class and they are capitalized

Reply:
oka i got it